core/src/banking_stage/vote_storage.rs

```diff
@@ -126,96 +62,31 @@ impl VoteStorage {
         )
     }
 
-    pub fn drain_unprocessed(&self, bank: &Bank) -> Vec<Arc<ImmutableDeserializedPacket>> {
-        self.latest_unprocessed_votes.drain_unprocessed(bank)
-    }
-
-    // returns `true` if the end of slot is reached
-    pub fn process_packets<F>(
+    // Re-insert re-tryable packets.
+    pub(crate) fn reinsert_packets(
         &mut self,
-        bank: &Bank,
-        banking_stage_stats: &BankingStageStats,
-        slot_metrics_tracker: &mut LeaderSlotMetricsTracker,
-        mut processing_function: F,
-    ) -> bool
-    where
-        F: FnMut(
-            usize,
-            &mut bool,
-            &mut Vec<RuntimeTransaction<SanitizedTransaction>>,
-            &mut LeaderSlotMetricsTracker,
-        ) -> Option<Vec<usize>>,
-    {
-        if matches!(self.vote_source, VoteSource::Gossip) {
-            panic!("Gossip vote thread should not be processing transactions");
-        }
```

Comment:
noting that this was removed and is fine because `process_buffered_packets` checks `vote_storage.should_not_process`